rebasehelper/versioneer.py

```diff
@@ -70,20 +70,22 @@ def get_available_versioneers(self):
         """Returns a list of available versioneers"""
         return [k for k, v in six.iteritems(self.versioneers)]
 
-    def run(self, versioneer, package_name, category):
+    def run(self, versioneer, package_name, category, disabled_versioneers):
         """
         Runs specified versioneer or all versioneers subsequently until one of them succeeds.
 
         :param versioneer: Name of a versioneer
         :param package_name: Name of a package
         :param category: Package category
+        :param disabled_versioneers: Versioneers disabled on the command line or in a config file
```

Comment:
This is not accurate, what if at some point this method will be run from some other location and with a set of disabled versioneers that is acquired from a totally different source (or just always empty for instance)?
You should state what is the meaning of this argument in context of this method, so for example: `List of versioneers that will be skipped`.